e2e/cypress/integration/onboarding/login_page_link_account_creation_spec.js

```diff
@@ -0,0 +1,137 @@
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @onboarding
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import {getEmailUrl, getEmailMessageSeparator, reUrl, getRandomId} from '../../utils';
+
+describe('Onboarding', () => {
+    let testTeam;
+    const randomId = getRandomId();
+    const username = `user${randomId}`;
+    const email = `user${randomId}@sample.mattermost.com`;
+    const password = 'passwd';
+
+    const baseUrl = Cypress.config('baseUrl');
+    const mailUrl = getEmailUrl(baseUrl);
+
+    before(() => {
+        // # Disable LDAP, require email invitation, and do email test if setup properly
+        cy.apiUpdateConfig({LdapSettings: {Enable: false}, EmailSettings: {RequireEmailVerification: true}});
+        cy.apiEmailTest();
+
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T400 Create account from login page link using email-password', () => {
+        cy.get('.sidebar-header-dropdown__icon').click();
+        cy.findByText('Team Settings').should('be.visible').click();
+
+        // * Check that the 'Team Settings' modal was opened
+        cy.get('#teamSettingsModal').should('exist').within(() => {
+            cy.get('#open_inviteEdit').should('be.visible').click();
+
+            // # Enable any user with an account on the server to join the team
+            cy.get('#teamOpenInvite').should('be.visible').click();
+            cy.findByText('Save').should('be.visible').click();
+
+            // # Close the modal
+            cy.get('#teamSettingsModalLabel').find('button').should('be.visible').click();
+        });
+
+        // * In system console, verify that 'Enable Open Server' is set to true
+        cy.get('.sidebar-header-dropdown__icon').click();
+        cy.get('#systemConsole').should('be.visible').click();
+        cy.findByText('Signup').scrollIntoView().should('be.visible').click();
+        cy.get(`#${CSS.escape('TeamSettings.EnableOpenServertrue')}`).should('be.checked');
+
+        // # Logout from sysadmin account
+        cy.apiLogout();
+
+        // # Visit the team url
+        cy.visit(`/${testTeam.name}`);
+
+        // # Attempt to create a new account
+        cy.get('#signup', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible').click();
```

Comment:
This one failed on me.

Reply:
It was also flaky for me, so I added a new `FORTY_FIVE_SEC` in `timeouts.js`, and now it is consistently passing for me. I also moved the timeout to a new visibility check for `#login_section`.. I hope it is okay, otherwise I can find another way :)

Reply:
I could of course use `ONE_MIN` but I felt that it's quite excessive, but since it's a `timeout` and not an explicit wait, that could also work I think.

Reply:
Yup, it's fine. If it's an explicit wait, then that will definitely expensive :)

Reply:
Thanks @saturninoabril :)